concourse/model/traits/release.py

```diff
@@ -48,6 +48,15 @@
         ''',
         type=bool,
     ),
+    AttributeSpec.optional(
+        name='dev_cycle_callback',
```

Comment:
how about naming this 'next_version_callback' (or next_version_callback_path)? Would that maybe be more intuitive? At least I would not understand the meaning of 'dev_cycle_callback' (I was unhappy with out internal variable names there, before)

Reply:
Thanks, I also found the name to be a bit counter-intuitive. I went with your proposal and changed it to `next_version_callback`.